diff --git a/Backend/src/modules/lender/lender.service.spec.ts b/Backend/src/modules/lender/lender.service.spec.ts
index b0bb4dd..2ba03e5 100644
--- a/Backend/src/modules/lender/lender.service.spec.ts
+++ b/Backend/src/modules/lender/lender.service.spec.ts
@@ -1,12 +1,25 @@
 import { Test, TestingModule } from '@nestjs/testing';
 import { LenderService } from './lender.service';
+import { FirebaseService } from '../../firebase/firebase.service';
 
 describe('LenderService', () => {
   let service: LenderService;
 
+  const mockFirebaseService = {
+    db: {
+      collection: jest.fn(),
+    },
+  };
+
   beforeEach(async () => {
     const module: TestingModule = await Test.createTestingModule({
-      providers: [LenderService],
+      providers: [
+        LenderService,
+        {
+          provide: FirebaseService,
+          useValue: mockFirebaseService,
+        },
+      ],
     }).compile();
 
     service = module.get<LenderService>(LenderService);
